domain/model/modelmigration/import.go

```diff
@@ -183,7 +183,7 @@ func (i *importOperation) Execute(ctx context.Context, model description.Model)
 	}
 
 	args := domainmodel.ModelImportArgs{
-		ModelCreationArgs: domainmodel.ModelCreationArgs{
+		ControllerDBModelCreationArgs: domainmodel.ControllerDBModelCreationArgs{
```

Comment:
Is the naming of this correct? Maybe we just don't compose these args together, and just have `ModelImportArgs`, otherwise it feels like you're creating a controller model here, which isn't possible.